pkg/lib/localstore/objectstore/notification_store.go

```diff
@@ -0,0 +1,57 @@
+package objectstore
+
+import (
+	"github.com/dgraph-io/badger/v4"
+	"github.com/gogo/protobuf/proto"
+	ds "github.com/ipfs/go-datastore"
+
+	"github.com/anyproto/anytype-heart/pkg/lib/localstore"
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
+	"github.com/anyproto/anytype-heart/util/badgerhelper"
+)
+
+const notificationsPrefix = "notifications"
+
+var notificationsInfo = ds.NewKey("/" + notificationsPrefix + "/info")
+
+type NotificationStore interface {
```

Comment:
maybe move this to notification service?

Reply:
i mean extract notification-related code to service. Methods in objectstore that don't related to objects are legacy

Reply:
Extract it to separate component